### DISCUSSION TITLE
Can't active extension behind Cloudflare teams

### DISCUSSION THREAD

**Discussion by chunlea** [community/community#8804] [Copilot Conversations] [answered]
I am currently using cloudflare wrap and our company using cloudflare teams as our VPN solution. I got this error and copilot not working. 

```
[INFO] [default] [2021-12-13T20:33:40.748Z] Experimentation service initialized
[INFO] [default] [2021-12-13T20:33:40.861Z] Feature for copilotaa: true
[INFO] [default] [2021-12-13T20:33:40.861Z] Feature for copilotincludeprompt: none
[ERROR] [default] [2021-12-13T20:33:41.019Z] GitHub Copilot could not connect to server. Extension activation failed: "unable to get local issuer certificate"
```

**Comment by davecheney** [comment, marked as answer]
I'm sorry Copilot does not work through proxies or VPNs which intercept your SSL traffic.

**Reply by veb** [reply]
Is that correct though? Using an extension like `win-ca` for Windows (https://marketplace.visualstudio.com/items?itemName=ukoloff.win-ca) and then playing with replace/append. 

My old work had a firewall that issued its own certificate (effectively MITM'ing me) but that helped immensely. So @chunlea could possibly give that a try.

Oh! Looks like there's a version for Mac too: https://github.com/jfromaniello/mac-ca

Could be worth a try :)

**Reply by ArthurBPinotti** [reply]
it worked for me, thanks

**Reply by J4NN1K-E** [reply]
how did  you do that?
